### ISSUE TITLE
Make filters on Student Directory and Related Programs focusable

### ISSUE THREAD

**Issue by jeremyboggs** [praxisnetwork/praxisnetwork.github.com#68]
The filters on the the Student Directory and Related Programs are not accessible using keyboard navigation.
